### PR TITLE
Made changes in the Team Cards UI

### LINKED ISSUE #279: Improving the UI of cards in teams page
## Is your feature request related to a problem? 

It is not problem, but the colors that are used for cards (just grey for the text)  in teams page doesn't appear striking as the contrast is too low. Even the icons are of the same color. The font size could also be increased for better readability. 

## **Describe the solution you'd like**

Increasing the font size and using the the primary color of the website for icons and the names of the team members. Also, improve the look of the cards. 
I would like to work on this issue.

### PR DESCRIPTION
**Pull request by harshb02**
# Description

Changed the color of the name and description of the team members and added some styles. Also added a hover effect on the social icons.
Fixes #279

## List any dependencies that are required for this change 
NA

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?
Ran on local server
## UI /UX changes
   Attach gif or screenshot for changes.
   - Before:
   
screencapture-file-C-Users-HARSH-Downloads-Algo-Phantoms-Frontend-master-2-Algo-Phantoms-Frontend-master-team-html-2021-03-29-20_49_22

- After:

screencapture-file-C-Users-HARSH-Desktop-AlgoPhantom-Algo-Phantoms-Frontend-team-html-2021-03-29-20_49_33

demo

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -343,28 +343,70 @@ hr {
...
+  .socials a:hover {
+    color: #3caea3;
+    box-shadow: 0 0 5px #3caea3;
+    text-shadow: 0 0 5px #3caea3;
+  }
+
+
 .box p {
-    color: #ffffff90;
+    color: white;
```
Comment (harikesh409) [inline css/style.css:400]: Use hex color.

File: css/style.css [hunk 2]
```diff
@@ -343,28 +343,70 @@ hr {
...
 
 .socials {
     margin-top: 35px;
 }
 
-.socials i {
-    margin: 0 10px;
-    font-size: 25px;
-}
+
```
Comment (harikesh409) [inline css/style.css:409]: Remove the unnecessary empty lines.

**Review by harikesh409 — CHANGES_REQUESTED** [review]

**Review by harikesh409 — CHANGES_REQUESTED** [review]
image
The name **MANSHI TODI** is no visible properly due the color. So just move that circle background a bit top so that the name is visible.

**Comment by harshb02** [comment]
> image
> The name **MANSHI TODI** is no visible properly due the color. So just move that circle background a bit top so that the name is visible.

The changes have been made kindly review them @harikesh409

**Comment by harshb02** [comment]
We both have made changes in the teams page and have merged our ideas. She has already made a PR of the changes she has done (issue #314). Kindly review the changes @harikesh409

**Comment by Khushi-agarwal** [comment]
@harikesh409  we have made the changes as requested by you in the pr which I created that is pr #314 and all the changes have been added here. So kindly review it.

**Comment by Khushi-agarwal** [comment]
Also @harikesh409  this pr is not assigned to me. So please assign it to me as well.

**Review by ravikr126 — APPROVED** [review]

**Comment by Khushi-agarwal** [comment]
@ravikr126 please assign this pr to me as well as I have also made the changes.

**Review by harikesh409 — CHANGES_REQUESTED** [review]
All changes are looking good but the `team.html` page is not upto date with the master branch check the navbar few options are missing kindly pull code from master and merge your changes then we can merge the branch.

**Comment by harshb02** [comment]
I have updated the team.html page kindly check @harikesh409 @Khushi-agarwal

**Pull request merged**